lib/lichess.py

```diff
@@ -2,7 +2,7 @@
 import json
 import requests
 from urllib.parse import urljoin
-from requests.exceptions import ConnectionError, HTTPError, ReadTimeout
+from requests.exceptions import ConnectionError as RequestsConnectionError, HTTPError, ReadTimeout
```

Comment:
Why is this renamed?

Reply:
Because of the error `"A004" # Import is shadowing a Python builtin`. Python has a `ConnectionError` itself which can be used without being imported, so by importing the new error from `requests` we overrode it.